Flick/Controllers/ListViewController.swift

```diff
@@ -68,22 +70,51 @@ class ListViewController: UIViewController {
         mediaCollectionView.bounces = false
         view.addSubview(mediaCollectionView)
 
+        // Empty list
+        addMediaMessageLabel.text = "Nothing here yet. Add\nsome movies or shows!"
+        addMediaMessageLabel.textColor = .darkBlue
+        addMediaMessageLabel.textAlignment = .center
+        addMediaMessageLabel.font = .systemFont(ofSize: 18, weight: .medium)
+        addMediaMessageLabel.numberOfLines = 0
+
+        arrowToAddButtonView.image = UIImage(named: "arrowToButton")
+        emptyListImageView.image = UIImage(named: "emptyList")
+
+        if list.shows.count == 0 {
+            view.addSubview(emptyListImageView)
```

Comment:
I think we should be able to move all the initialization of emptyListImageView, addMediaMessageLabel, and arrowToAddButtonView into this "if list.shows.count"? cause we only use if the list is empty right? so maybe even have a separate setUpEmptyView() and call it if list.shows.count == 0 ?

Reply:
good point